lib/facebook.ex

```diff
@@ -221,62 +221,86 @@ defmodule Facebook do
   end
 
   @doc """
-  Gets the total number of people who liked an object.
+  Gets the number of elements that a scope has in a given object.
+
   An *object* stands for: post, comment, link, status update, photo.
 
-  If you want to get the likes of a page, please see *pageLikes*.
+  If you want to get the likes of a page, please see *fanCount*.
+
+  Expected scopes:
+    * :likes
+    * :comments
 
   ## Example
       iex> Facebook.objectCount(:likes, "1326382730725053_1326476257382367", "<Token>")
       2
+      iex> Facebook.objectCount(:comments, "1326382730725053_1326476257382367", "<Token>")
+      2
 
   See: https://developers.facebook.com/docs/graph-api/reference/object/likes
+  See: https://developers.facebook.com/docs/graph-api/reference/object/comments
   """
-  def objectCount(:likes, object_id, access_token) do
-    "likes"
-      |> objectSummary(object_id, access_token)
+  def objectCount(scope, object_id, access_token) when is_atom(scope) do
+    params = [access_token: access_token, summary: true]
+    if !is_nil(Config.appsecret) do
+      params = params ++ [appsecret_proof: encrypt(access_token)]
+    end
+
+    scp = scope
+      |> Atom.to_string
+      |> String.downcase
+
+    Facebook.Graph.get(~s(/#{object_id}/#{scp}), params)
+      |> getSummary
       |> summaryCount
   end
 
   @doc """
-  Gets the total number of people who commented an object.
-  An *object* stands for: post, comment, link, status update, photo.
+  Gets the number of reactions that an object has.
 
-  ## Example
-      iex> Facebook.objectCount(:comments, "1326382730725053_1326476257382367", "<Token>")
-      2
+  Expected type of reactions:
+    * :haha
+    * :wow
+    * :thankful
+    * :sad
+    * :angry
+    * :love
+    * :none
 
-  See: https://developers.facebook.com/docs/graph-api/reference/object/comments
+  ## Examples
+      iex> Facebook.objectCount(:reaction, :wow, "769860109692136_1173416799336463", "<Token>")
+      2
+      iex> Facebook.objectCount(:reaction, :haha, "769860109692136_1173416799336463", "<Token>")
+      12
+      iex> Facebook.objectCount(:reaction, :thankful, "769860109692136_1173416799336463", "<Token>")
+      33
   """
-  def objectCount(:comments, object_id, access_token) do
-    "comments"
-      |> objectSummary(object_id, access_token)
+  def objectCount(:reaction, react_type, object_id, access_token) when is_atom(react_type) do
```

Comment:
if you generate the docs locally using `mix docs` and open `doc/index.html`, the objectCount for reactions is outlined with the first param being `atom1`. Not sure what to do about it. Could an `@spec` annotation help? Do you know that?